### PR TITLE
Update adapter-fenics.md

### PR DESCRIPTION
**Pull request by ajaust**
Update the FEniCS-preCICE literature after paper has been published.

### CODE REVIEW

File: pages/docs/adapters/fenics/adapter-fenics.md [hunk 1]
```diff
@@ -42,5 +42,6 @@ If you are using our adapter, please consider citing our paper "FEniCS-preCICE:
 
 ## Related literature
 
-[1] Benjamin Rodenberg, Ishaan Desai, Richard Hertrich, Alexander Jaust, Benjamin Uekermann. FEniCS-preCICE: Coupling FEniCS to other Simulation Software. [preprint on arXiv, arxiv.org/abs/2103.11191](https://arxiv.org/abs/2103.11191), 2021  
+[1] Benjamin Rodenberg, Ishaan Desai, Richard Hertrich, Alexander Jaust, Benjamin Uekermann. FEniCS-preCICE: Coupling FEniCS to other Simulation Software. SoftwareX, Volume 16, 2021, [doi:10.1016/j.softx.2021.100807](https://doi.org/10.1016/j.softx.2021.100807), [preprint on arXiv](https://arxiv.org/abs/2103.11191)
```
Comment (davidscn) [inline pages/docs/adapters/fenics/adapter-fenics.md]: Do you intentionally want to keep the link to the preprint? The actual publication is anyway accessible, right?

Comment (ajaust) [reply pages/docs/adapters/fenics/adapter-fenics.md]: That is a good point. I had removed it first and then added it again... I guess it is save to remove it as the paper is open access. I just removed the link to the preprint.

**Comment by ajaust** [comment]
We just discussed the citation information of the adapter in https://github.com/precice/fenics-adapter/issues/140. I will close the current issue here and create a new one with a bigger rework of the citation information.

**Comment by BenjaminRodenberg** [comment]
I think we could still merge this PR. Generally it's a step into the right direction. Feel free to reopen and merge.

**Comment by ajaust** [comment]
Then lets merge this one for now and I will change it further afterwards.

**Pull request closed without merging**